templates/my_study/transform_module.py

```diff
@@ -48,8 +49,14 @@ def transform_function(mapped_df_dict):
     }
 
     """
-
-    df = list(mapped_df_dict.values())[0]
+    df = mapped_df_dict['extract_config.py']
+
+    # df = outer_merge(
```

Comment:
Why are these lines commented out? Isn’t this what we want to make the template package functional. Or are we assuming someone will step through the tutorial and uncomment these lines? If the latter, then this needs to be made clearer in the tutorial

Reply:
> Why are these lines commented out?

Because they're not trash, but they also break the module if you try to test extract before getting to the transform instructions.

>  Isn’t this what we want to make the template package functional.

Only for the transform section, which is why it's still there as a comment.